client/package.json

```diff
@@ -74,7 +73,8 @@
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-react": "^7.19.0",
     "prettier": "2.0.5",
-    "react-inspector": "^3.0.2"
+    "react-inspector": "^3.0.2",
+    "typescript": "^4.0.2"
```

Comment:
We are using typescript now?

Reply:
I think this must have been a to resolve a peer dependency warning from npm.
We're not using it directly, and everything installs fine without it, so I'll remove it.

Reply:
I think this must have been a to resolve a peer dependency warning from npm.
We're not using it directly, and everything installs fine without it, so I'll remove it.